docs/get-started/configuration.md

```diff
@@ -0,0 +1,22 @@
+---
+id: configuration
+title: Configuration
+sidebar_label: Configuration
+---
+
+There are two files you can configure right now: `webiny.root.js` and `webiny.config.js`
+
+## webiny.root.js
+
+This file appears in your project's root and it is unique. These are the configurable fields:
+
+- **projectName** - your project's name; it is passed into the **Context** the plugins receive and it is also used when delpoying to AWS.
+- **cli**
+    -  **plugins** - an array of CLI and Scaffold plugins that are usable in our CLI
```

Comment:
Maybe an example of a plugin, or a link to something that already references how to create a plugin?